README.md

```diff
@@ -178,15 +180,117 @@ services:
 - step 8: Run `docker-compose up -d`
 - step 9: This might take a while if `docker-compose logs op-public-consumer |grep success Returns: Initial sync http://redpencil.data.gift/id/job/URI has been successfully run`; you should be good. (Your computer will also stop making noise)
 
+#### gn-publications-consumer
+
+##### 1. Initial sync
+
+##### From scratch
+
+Setting up the sync should happen work with the following steps:
+
+- Ensure `docker-compose.override.yml` has AT LEAST the following information:
+
+```yml
+version: '3.7'
+
+services:
+#(...) there might be other services
+
+  gn-publications-consumer:
+    environment:
+      DCR_SYNC_BASE_URL: "https://gn-harvester.s.redhost.be/" # You can choose the endpoint here.
+      DCR_DISABLE_DELTA_INGEST: "true"
+      DCR_DISABLE_INITIAL_SYNC: "true"
+# (...) there might be other information
+```
+
+- Start the stack: `drc up -d` and ensure the migrations have run and finished `drc logs -ft --tail=200 migrations`
+- The sync can now be started; update the following in `docker-compose.override.yml`:
+
+```yml
+version: '3.7'
+
+services:
+#(...) there might be other services
+
+  gn-publications-consumer:
+    environment:
+      DCR_SYNC_BASE_URL: "https://gn-harvester.s.redhost.be/" # You can choose the endpoint here.
+      DCR_DISABLE_DELTA_INGEST: "false" # <------ THIS CHANGED
+      DCR_DISABLE_INITIAL_SYNC: "false" # <------ THIS CHANGED
+# (...) there might be other information
+```
+
+- Start the sync `drc up -d gn-publications-consumer`.
+  Data should be ingesting.
+  Check the logs `drc logs -ft --tail=200 gn-publications-consumer`
+- This step might take a while. If `drc logs gn-publications-consumer | grep success` returns: `Initial sync http://redpencil.data.gift/id/job/URI has been successfully run`, things should be good to go.
+
+##### In case of a re-sync
+
+In some cases, you may need to reset the data due to unforeseen issues. The simplest method is to entirely flush the triplestore and start from a fresh slate. However, this can be time-consuming, and if the app possesses an internal state that cannot be recreated from external sources, a more granular approach would be necessary. We will outline this approach here.
+
+- Step 1: Ensure the app is running, and all migrations have run.
+- Step 2: Ensure `gn-publications-consumer` has stopped syncing; `docker-compose.override.yml` should AT LEAST contain the following information:
+```yml
+version: '3.7'
+
+services:
+#(...) there might be other services
+
+  gn-publications-consumer:
+    environment:
+      DCR_DISABLE_DELTA_INGEST: "true"
+      DCR_DISABLE_INITIAL_SYNC: "true"
+     # (...) there might be other information e.g. about the endpoint
+
+# (...) there might be other information
+```
+
+- Step 3: `drc up -d gn-publications-consumer` to re-create the container.
+- Step 4: We need to flush the ingested data. Sample migrations have been provided.
+
+```bash
```

Comment:
This should need an adaptation I think, because the procedure is only valid for besluiten. I think you can steal the migration-template from burgernabije databank

Reply:
Yes, the sample migrations were a part I wanted to ask about. Is there a data model I can use to adapt the sample migrations to the GN publications?

Reply:
The model and graph are exactly the same, so i think you can really copy/past this folder.
https://github.com/lblod/app-burgernabije-besluitendatabank/tree/master/config/sample-migrations